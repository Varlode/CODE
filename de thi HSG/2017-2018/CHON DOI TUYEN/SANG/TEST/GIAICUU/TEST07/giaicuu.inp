3
9 9
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
9 9 999999
9 9
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
9 9 99999
9 9
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
999 999 999 999 999 999 999 999 999
9 9 9999