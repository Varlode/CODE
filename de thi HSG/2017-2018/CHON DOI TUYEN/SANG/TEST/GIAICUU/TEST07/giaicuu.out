YES
983016
YES
83016
NO